lib/mix/tasks/nlp/pos_tagger/test.ex

```diff
@@ -0,0 +1,93 @@
+defmodule Mix.Tasks.Nlp.PosTagger.Test do
```

Comment:
this is really cool!

Reply:
Thanks. I was going to supply just the train task with optional testing, but then I realized running several tests over different datasets might be desirable, and you obviously don't want to retrain each time.